java8-maven/src/main/java/EmitLog.java

```diff
@@ -0,0 +1,35 @@
+import com.rabbitmq.client.BuiltinExchangeType;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.Channel;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class EmitLog {
+
+    private static final String EXCHANGE_NAME = "logs";
+
+    public static void main(String[] argv) throws Exception {
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setHost("localhost");
+        try (Connection connection = factory.newConnection();
+             Channel channel = connection.createChannel()) {
+
+            channel.exchangeDeclare(EXCHANGE_NAME, BuiltinExchangeType.FANOUT);
+
+            String message = getMessage(argv);
+
+            channel.basicPublish(EXCHANGE_NAME, "", null, message.getBytes("UTF-8"));
+            System.out.println(" [x] Sent '" + message + "'");
+        }
+    }
+
+    private static String getMessage(String[] strings) {
+        if (strings.length < 1)
+            return "info: Hello World!";
+        return Stream.of(strings).collect(Collectors.joining(" "));
```

Comment:
I like `Stream`s too, but `String#join` would be more appropriate, wouldn't it? (and it's Java 8 too!)

Reply:
I agree with you. I will change them